classes/class-wp-stream-query.php

```diff
@@ -166,7 +166,7 @@ public function query( $args ) {
 
 		// PARSE PROPERTIES
 		foreach ( $properties as $property => $default ) {
-			if ( $args[ $property ] ) {
+			if ( isset( $args[ $property ] ) && '' !== $args[ $property ] && false !== $args[ $property ] && ! is_array( $args[ $property ] ) ) {
```

Comment:
I've also opted to totally ignore array values altogether since they should never be allowed in single property values, only `__in` and `__not_in` property variants should have array values.